lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -190,6 +188,15 @@ def _upload(self, context, success_rows, error_rows):
         if len(error_rows) > 1:
             upload_csv_to_report_store(error_rows, context.file_name + '_err', context.course_id, date)
 
+    def log_additional_info_for_mit_course(self, context, message):
```

Comment:
Where is the course key check added?

Reply:
if `log_additional_info` is True, we would log the information. The course key check is inside the ProblemGradeReportContext class variable. `log_additional_info`